lib/presentation/screens/auth/register.dart

```diff
@@ -0,0 +1,234 @@
+                          onPressed: () {
+                            final email = _emailController.text.trim();
+                            final username = _usernameController.text.trim();
+                            final password = _passwordController.text;
+                            final confirmPassword =
+                                _confirmPasswordController.text;
+
+                            if (email.isEmpty ||
+                                username.isEmpty ||
+                                password.isEmpty ||
+                                confirmPassword.isEmpty) {
+                              Get.snackbar(
+                                "Registrasi Gagal",
+                                "Semua data harus diisi",
+                                snackPosition: SnackPosition.TOP,
+                                colorText: Colors.white,
+                              );
+                              return;
+                            }
+
+                            if (password != confirmPassword) {
+                              Get.snackbar(
+                                "Registrasi Gagal",
+                                "Kata sandi tidak valid",
+                                snackPosition: SnackPosition.TOP,
+                                colorText: Colors.white,
+                              );
+                              return;
+                            }
+
+                            Get.snackbar(
+                              "Registrasi Berhasil",
+                              "Akun berhasil didaftarkan",
+                              snackPosition: SnackPosition.TOP,
+                              colorText: Colors.white,
+                            );
+
+                            Get.toNamed('/signin');
+                          },
+                          style: ElevatedButton.styleFrom(
+                            backgroundColor:
+                                HexColor("3E2F9A").withOpacity(0.45),
+                            foregroundColor: Colors.white,
+                            shape: RoundedRectangleBorder(
+                              borderRadius: BorderRadius.circular(24),
+                            ),
+                            side: const BorderSide(
+                              color: Colors.white,
+                              width: 0.5,
+                            ),
+                            padding: const EdgeInsets.symmetric(
+                              vertical: 20,
+                            ),
+                          ),
+                          child: const Text(
+                            "Daftar",
+                            style: TextStyle(fontSize: 16),
+                          ),
+                        ),
+                      ),
+                      const SizedBox(
+                        height: 28,
+                      ),
+                    ],
+                  )),
```

Comment:
yg double kayak ini di fix formatter nyo, kasih coma biar auto fix
```suggestion
                  ),),
```